src/types.ts

```diff
@@ -0,0 +1,13 @@
+export type DB = {
+  credentials: Credential[];
+};
+
+export type Credential = {
+  credentials: [
+    {
+      service: string;
+      username: string;
+      password: string;
+    }
+  ];
+};
```

Comment:
This is not entirely correct, DB would be an array of objects with the property "credentials" containing an array of objects with `service`, `username` and `password`. You should remove the encapsulating object and array